Description: Make the build reproducible
Author: Chris Lamb <lamby@debian.org>
Last-Update: 2025-01-16
Forwarded: https://github.com/RsyncProject/rsync/pull/791

--- rsync-3.3.0+ds1.orig/md-convert
+++ rsync-3.3.0+ds1/md-convert
@@ -247,6 +247,9 @@ def find_man_substitutions():
 
     env_subs['date'] = time.strftime('%d %b %Y', time.gmtime(mtime + tz_offset)).lstrip('0')
 
+    if 'SOURCE_DATE_EPOCH' in os.environ:
+        env_subs['date'] = time.strftime('%d %b %Y', time.gmtime(int(os.environ.get('SOURCE_DATE_EPOCH', time.time()))))
+
 
 def html_via_commonmark(txt):
     return commonmark.HtmlRenderer().render(commonmark.Parser().parse(txt))
